spinalcordtoolbox/gui/base.py

```diff
@@ -283,6 +285,29 @@ def __init__(self, image, params, init_values=None):
         self.params = params
         self.points = []
         self._overlay_image = init_values
+        self.setup_intensity()
+
+    def setup_intensity(self):
+        if self.params.vmode == 'percentile':
+            vol = self.image.flatten()
+            vol = vol[vol > 0]
+            first_per = np.percentile(vol, self.params.vmin)
+            last_per = np.percentile(vol, self.params.vmax)
+            self.mean_intensity = np.percentile(vol, self.params.vmean)
+            self.std_intensity = last_per - first_per
```

Comment:
I don't think these should be called `mean/std`, it's a bit misleading.

Reply:
Also It doesn't look like there's a point for them to be stored in the object, they aren't used.

Reply:
Can you give more detail what is misleading?